addOns/reports/src/main/java/org/zaproxy/addon/reports/sarif/SarifToolData.java

```diff
@@ -0,0 +1,201 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2021 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.reports.sarif;
+
+public class SarifToolData {
+
+    public static final SarifToolData INSTANCE = new SarifToolData();
+
+    // @formatter:off
```

Comment:
Remove these, this does not work.

Reply:
@thc202 : Please explain in more detail what you want to have changed/what exactly is not working.

Reply:
What psiinon said, remove those comments, they don't work, formatting happens always.

Reply:
Ah. Thank you. In our project we use the eclipse formatter together with spotless - here the formatter:on /off flags do work. Will remove all of those comments in code.

Reply:
Not all of them were removed yet e.g. `SarifReportDataSupportUnitTest`.